concepts/comparison/links.json

```
@@ -1 +1,14 @@
-[]
+[
+  {
+    "url": "https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators#relational_operators",
+    "description": "Links to the official documentation for all relational operators"
```

```suggestion
    "description": "MDN: Relational operators"
```

This makes these in-line with the other links. I'm also hesitant calling it official without denoting the authority (even tho I _feel_ the same way).